specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -2444,6 +2676,14 @@
             "required": true,
             "type": "string",
             "x-ms-parameter-location": "method"
+        },
+        "TopologyResourceName": {
+            "name": "topologyResourceName",
+            "in": "path",
+            "description": "Name of an topology resource group.",

```

Comment:
resource group?
isn't it just a resource?

Reply:
the term "resource group" is already in use so it might confuse customers, please change the description